extensions/transform/org.eclipse.smarthome.transform.scale.test/src/test/java/org/eclipse/smarthome/transform/scale/internal/ScaleTransformServiceTest.java

```diff
@@ -98,4 +98,16 @@ public void testTransformByScaleErrorInBounds() throws TransformationException {
 
     }
 
+    @Test
+    public void testEvaluationOrder() throws TransformationException {
+        // Ensures that only first matching scale as presented in the file is taken in account
+        String evaluationorder = "scale/evaluationorder.scale";
+        // This value matches two lines of the scale file
+        String source = "12";
+
+        String transformedResponse = processor.transform(evaluationorder, source);
+        Assert.assertEquals("first", transformedResponse);
+
```

Comment:
Maybe leave out this newline, note that it also is a notice in the (upcoming) release of the static code analysis.